Game/Config/DefaultEditorSpatialGDK.ini

```diff
@@ -8,52 +8,54 @@ TestSuiteGameStateBase=
 ; ClassName=include_path.h
 ; Leave empty if no includes required.
 
+/Game/ThirdPerson/TypebindingActor.TypebindingActor=
```

Comment:
Curious, why was this typebinding added?

Reply:
To test the feature

Reply:
We should add it to the bottom of the schema to prevent changing the `id` of the other schema components. It's read sequentially :p